docs/proton-kafka.md

```diff
@@ -41,57 +39,67 @@ SETTINGS
     properties='..'
 ```
 
-:::info
+### Settings
 
-For examples to connect to various Kafka API compatitable message platforms, please check [this doc](/tutorial-sql-connect-kafka).
+#### type
 
-:::
-### DDL Settings
+Must be set to `kafka`. Compatible with:
 
-#### type
-Need to be `kafka`. This works with Apache Kafka, Redpanda, Confluent Platform or Cloud,and many other Kafka compatible message platforms.
+* Apache Kafka
+* Confluent Platform or Cloud
+* Redpanda
+* Other Kafka-compatible systems
 
 #### brokers
-One or more brokers with ports.
+
+Comma-separated list of broker addresses (host\:port), e.g.:
+
+```
+kafka1:9092,kafka2:9092,kafka3:9092
+```
 
 #### topic
-One external stream will connect to one topic.
+
+Kafka topic name to connect to.
 
 #### security_protocol
+
 The supported values for `security_protocol` are:
 
 - PLAINTEXT: when this option is omitted, this is the default value.
 - SASL_SSL: when this value is set, username and password should be specified.
-  - If you need to specify own SSL certification file, add another setting `ssl_ca_cert_file='/ssl/ca.pem'` New in Proton 1.5.5, you can also put the full content of the pem file as a string in the `ssl_ca_pem` setting if you don't want to, or cannot use a file path, such as on Timeplus Enterprise or in Docker/Kubernetes environments.
-  - Skipping the SSL certification verification can be done via `SETTINGS skip_ssl_cert_check=true`.
+  - If users need to specify own SSL certification file, add another setting `ssl_ca_cert_file='/ssl/ca.pem'`. Users can also put the full content of the pem file as a string in the `ssl_ca_pem` setting.
+  - To skip the SSL certification verification: `skip_ssl_cert_check=true`.
 
 #### sasl_mechanism
+
 The supported values for `sasl_mechanism` are:
 
-- PLAIN: when you set security_protocol to SASL_SSL, this is the default value for sasl_mechanism.
+- PLAIN: when setting security_protocol to SASL_SSL, this is the default value for sasl_mechanism.
 - SCRAM-SHA-256
 - SCRAM-SHA-512
-- AWS_MSK_IAM: this is available since Timeplus Enterprise 2.7.0 and Proton 1.6.12. Set to this value if you are using AWS MSK with IAM authentication and the EC2 instance or Kubernetes pod has the proper IAM role to access the Kafka topic.
```

Comment:
We can remove the first sentence regarding version, but maybe keep the second one, since "AWS MSK IAM role-based access" is too abstract